development_environments/using_selenium_with_vagrant.md

```diff
@@ -0,0 +1,83 @@
+## [Home](../README.md)
+
+# Using Selenium with Vagrant
+The Selenium web driver for Capybara makes it easy to write integration tests because you can see and interact with the web browser that is running your tests. A downside of Selenium is it runs slower than headless drivers like PhantomJS. So a common approach is to use Selenium while you're writing or debugging tests, and then switch to a faster headless driver after the test is working.
+
+Vagrant works great for setting up isolated development environments, but since Vagrant runs in headless mode, it doesn't work well with Selenium.
+
+Here's how to make them play nicely together.
+
+## Setup
+
+### 1. Install Dependencies on Mac
+```
+$ brew install chromedriver
+$ brew install selenium-server-standalone
+```
+
+### 2. Configure Capybara and VCR
+Add this code to your test suite's config file (e.g. `features/env.rb`).
+
+```ruby
+SELENIUM_SERVER = '10.0.2.2' # found with `netstat -rn | grep "^0.0.0.0 " | cut -d " " -f10`
+SELENIUM_PORT = 4444
+RAILS_APP_HOST = '127.0.0.1:3001'
+
+# Register new driver
+Capybara.register_driver :selenium_remote do |app|
+  Capybara::Selenium::Driver.new(
+    app,
+    browser: :remote,
+    url: "http://#{SELENIUM_SERVER}:#{SELENIUM_PORT}/wd/hub",
```

Comment:
Is there value in splitting the host and port?  I think the Vagrant host will always be `10.0.2.2`